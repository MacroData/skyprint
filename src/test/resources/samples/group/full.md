# Group Messages
This is the first group of resources in this document. It is **recognized** by
the **keyword `group`** and its name is `Messages`.

## My Message [/message]

### Retrieve a Message [GET]
This action has **two** responses defined

+ Response 200 (text/plain)

        Hello World!
