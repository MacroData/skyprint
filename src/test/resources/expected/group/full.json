{
  "identifier": "Messages",
  "description": "This is the first group of resources in this document. It is **recognized** by the **keyword `group`** and its name is `Messages`.",
  "resources": [
    {
      "identifier": "My Message",
      "template": "/message",
      "method": null,
      "description": "",
      "actions": [
        {
          "identifier": "Retrieve a Message",
          "template": null,
          "method": "GET",
          "description": "This action has **two** responses defined"
        }
      ]
    }
  ]
}